include/networkit/centrality/DegreeCentrality.hpp

```diff
@@ -21,8 +21,9 @@ class DegreeCentrality: public Centrality {
 public:
     /**
      * Constructs the DegreeCentrality class for the given Graph @a G. If the centrality scores should be normalized,
-     * then set @a normalized to <code>true</code>. The run() method runs in O(m) time, where m is the number of
-     * edges in the graph.
+     * then set @a normalized to <code>true</code>. The run() method runs in O(n) time, where n is the number of
+     * nodes in the graph if there are no selfloops or selfloops are not being ignored. If they are being ignored
+     * and there are self loops in the graph then it runs in O(m) where m is the number of edges.
```

Comment:
Maybe simpler: run() runs in O(n) time if ignoreSelfLoops is false or the graph has no self-loops; otherwise it runs in O(m).